src/components/Image.js

```diff
@@ -104,7 +104,7 @@ export const Image = ({
         placeholderStyle={styles.placeholderStyle}
         accessible={!!sourceProp?.captionText}
         accessibilityLabel={`${sourceProp.captionText ? sourceProp.captionText : ''} ${
-          consts.a11yLabel.image
+          device.platform === 'ios' && consts.a11yLabel.image
```

Comment:
for `captionText` we have `...? ... : ''` and now you have used `&&`, why? could it be possible for some android, that TalkBack reads the `undefined`? i suggest having the same way of handling conditions for the same prop here and go for `...? ... : ''`, as we can be sure to have an empty string, that will not be read.

Reply:
Yeah, you're right. I changed it.

https://github.com/smart-village-solutions/smart-village-app-app/commit/f77c9eb27445cfab59a71cdb207e97cdd743d320